psydac/fem/tensor.py

```diff
@@ -312,6 +341,14 @@ def local_domain( self ):
         """
         return self._element_starts, self._element_ends
 
+    @property
+    def global_element_starts( self ):
+        return self._global_element_starts
+
+    @property
+    def local_element_ends( self ):
```

Comment:
Just like the previous property, this one should be called `global_element_ends`.

Reply:
thanks i forgot to change it